codecov.yaml

```diff
@@ -30,12 +27,21 @@ coverage:
         # swinging coverage that is not triggered by changes in a PR.
         # See: https://twistedmatrix.com/trac/ticket/10170
         threshold: 0.02%
+      tests:
+        # Make sure we run each test at least once.
+        # If we don't have 100% coverage for tests, it meant that some
+        # tests are always skipped.
+        target: 100%
+        paths:
+          - "src/incremental/tests/*"
```

Comment:
```suggestion
          - "src/incremental/tests/*"
          - "tests/*"
```

Reply:
Good comment. Thanks :)

But is the code from tests/* included in the coverage reporting?

This is the coveragerc file ... only the incremental main package is included
 
```diff
[run]
source =
+       incremental
branch = True

```

I see that in tox it runs multiple times.

I will see what I can do to fix it :)

Reply:
should be in

```
source_pkgs=incremental
source=tests
```

Reply:
I think that this will work as this is documented `source (multi-string): a list of packages or directories, `

but I agree that using source for directory and source_pkg for package is better.